### ISSUE TITLE
More samples

### ISSUE THREAD

**Issue by deman** [jlucansky/Camunda.Api.Client#18]
Hi,

It'd be very helpful if you could add more samples:
1. how to start a process
2. how to retrieve and complete task

Thank you.

**Comment by aabuabdou** [comment]
Indeed; I'd like to see a sample of how the FetchAndLock and then Complete work

**Comment by goremukin** [comment]
Indeed,

It is not clear how to complete the task

**Comment by DmitryKhi** [comment]
To complete a task, I added to ExternalTaskService.cs:
```C#
public Task Complete(ExternalTaskInfo externalTask, CompleteExternalTask completeExternalTask) => _api.Complete(externalTask.Id, completeExternalTask);
````

Sample code, which checks, fetches and complete tasks for a certain topic:
```C#
private const string WorkerName = "MyWorker";
private const string TopicName = "MyTopic";

// build query
var externalTaskQuery = new ExternalTaskQuery() { Active = true, TopicName = TopicName };

// add some sorting
externalTaskQuery
    .Sort(ExternalTaskSorting.TaskPriority, SortOrder.Descending)
    .Sort(ExternalTaskSorting.LockExpirationTime);

// request external tasks according to query
List<ExternalTaskInfo> tasks = await camunda.ExternalTasks.Query(externalTaskQuery).List();

if (tasks.Any())
{
    var fetchedTasks = await camunda.ExternalTasks.FetchAndLock(new FetchExternalTasks()
    {
        AsyncResponseTimeout = 60 * 1000,
        MaxTasks = 1,
        WorkerId = WorkerName,
        Topics = new List<FetchExternalTaskTopic>() { new FetchExternalTaskTopic(TopicName, 1000) }
    });

    Console.WriteLine($"Fetched tasks: {fetchedTasks.Count}");

    if (fetchedTasks != null && fetchedTasks.Count > 0)
    {
        foreach (var fetched in fetchedTasks)
        {
            // Consider the task completed
            Console.WriteLine($"Completing task: {fetched.Id}");

            CompleteExternalTask completeExternalTask = new CompleteExternalTask() { WorkerId = WorkerName };
            await camunda.ExternalTasks.Complete(fetched, completeExternalTask);
        }
    }
}
```

**Comment by bblair2018** [comment]
How do you start a process with variables, a business key and the name of the process?

**Comment by bblair2018** [comment]
I don't see the Complete method for the ExternalTasks Library. I am using Camunda.Api.Clinet version 2.6.0. Also, where is extend lock?

**Comment by gaensebluemchenritter** [comment]
Both reside in the ExternalTaskResource you get with ´camundaClient.ExternalTasks[id]´

**Comment by rahuliyer95** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
samajammin, @charmbrace, @ccccccmd, @jsheely, @ziyasal, @slthomason, @mengguang, @ahmetmeral, @Merix, @amemasire, @etuncay, @psytron, @moreal, @LarissaMaraSilvaFerreira, @jjfjjf2001, @rose-daniel, @ricvs, @timo991, @jason000000001, @ronin66666
